Dockerfile.svc

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7-alpine
+
+RUN apk add --update --no-cache alpine-sdk g++ gcc linux-headers libxslt-dev python3-dev build-base openssl-dev libffi-dev git && \
```

Comment:
needs `bash` as well - I think for whichever `gitpython` commands need the shell